pyproject.toml

```diff
@@ -67,7 +67,7 @@ docs = [
     'numpydoc>=1.7',
     'sphinx-copybutton',
     'matplotlib>=3.7',
-    'dask[array]>=2022.9.2',
+    'dask[array]>=2023.2.0',
```

Comment:
Guessing that this is done in accordance with [SPEC0](https://scientific-python.org/specs/spec-0000/)?